CHANGELOG.md

```diff
@@ -40,5 +40,24 @@ Before releasing:
 
 ### Removed
 
+## [0.5.0] - 2024-01-08
+
+### Added
+
+- Standard library like `Instant`s
+- Optical sensor bindings.
+- IMU sensor bindings.
+
+### Fixed
+
+- The async executor now does not starve the OS of cycles when unnecessary.
+
+### Changed
+
+- Updated readme with fixed grammar.
+
+### Removed
+
 [unreleased]: https://github.com/pros-rs/pros-rs/compare/v0.4.0...HEAD
 [0.4.0]: https://github.com/pros-rs/pros-rs/releases/tag/v0.4.0
+[0.5.0]: https://github.com/pros-rs/pros-rs/releases/tag/v0.5.0
```

Comment:
Would be great if you could update 0.5.0 here to point to `v0.4.0..v0.5.0` and unreleased to be `v0.5.0..HEAD`

Reply:
Also newer versions should be at the top to be consistent with [Keep a Changelog](https://keepachangelog.com/en/1.1.0/) and to keep people from having to scroll to see new releases